server/utils/dateUtils.test.ts

```diff
@@ -183,3 +192,17 @@ describe('dateIsBlank', () => {
     expect(dateIsBlank(date)).toEqual(true)
   })
 })
+
+describe('dateIsInThePast', () => {
+  it('returns true if the date is in the past', () => {
```

Comment:
Do we *have* to mock this? 
I hope 2020-01-01 consistently stays in the past

Reply:
Because we're mocking `isPast` for the other test, it makes sense to do it the other way. Also, not mocking would be more work, as we'd have to reimplement the default behaviour.